src/authorization/graphql/role.graphql

```diff
@@ -1,6 +1,7 @@
 type Role {
   id: ID!
   name: String!
+  permissions: [RolePermission]
```

Comment:
can't we keep array of permissions? [Permission]

Reply:
ok